client/src/containers/home/index.js

```diff
@@ -1,10 +1,10 @@
-import React, { Component } from "react";
-import { connect } from "react-redux";
-import logo from "../../logo.svg";
-import styles from "./index.css";
-import { createArticle } from "../../modules/article";
-import Spinner from "../spinner";
-import Links from "../Links";
+import React, { Component } from 'react';
```

Comment:
What code formatter do you use? I am not keen on having these kinds of back and forth changes. I use JSLint with no settings in vscode.

Reply:
I use prettier with eslint. Forgot to disable it before working on this project, it will be disabled in the future